### ISSUE TITLE
SQL comment throws !Unrecognized command: from console.sh xyz.osql command

### ISSUE THREAD

**Issue by dholtz** [orientechnologies/orientdb#5139]
Likely related to #4651
### Orientdb Version

2015-10-09 20:27:52:049 INFO  OrientDB Server v2.1.3 (build UNKNOWN@r; 2015-10-04 10:56:30+0000) is active. [OServer]

```
create class Person

create property Person.id string

create property Person.userName string

create property Person.password string

create class Manager extends Person

/* ManagerId */
```
### Error

!Unrecognized command: '/\* ManagerId */'

/usr/local/bin/orientdb-enterprise-2.1.3/bin $

**Comment by luigidellaquila** [comment]
Hi @dholtz 

comments are supported (2.1.x branch right now) inside statements, eg.

```sql
create class Manager extends Person /* ManagerId */
```
but the comment exclusion is made at parse level, while the console tries to interpret the command before the actual query parsing, this is why it's failing.
Flagged as a bug, but medium priority, I'll work on this as soon as I can

Thanks

**Comment by luigidellaquila** [comment]
I just added support for comments in the console, this will be released with v 3.0.3

Thanks

Luigi
